css-two-column-layout/styles.css

```diff
@@ -0,0 +1,77 @@
+body {
+  width: 80%;
+  margin: auto;
+}
+
+div {
+  justify-content: center;
+}
+
+.heading {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  width: 50%;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+  padding: 6rem;
+}
+
+.column {
+  display: flex;
+  margin: -2rem;
+}
+
+.column-text {
```

Comment:
Make sure you name `column` classes after their size, not what they will be applied to.